contrib/epee/include/storages/http_abstract_invoke.h

```diff
@@ -44,8 +47,11 @@ namespace epee
       if(!serialization::store_t_to_json(out_struct, req_param))
         return false;
 
+      http::fields_list additional_params;
+      additional_params.push_back(std::make_pair("Content-Type","application/json; charset=utf-8"));
```

Comment:
No, I think it's fine as that file is JSON specific. As long as we know why, it's good.